sources/packages/web/src/services/http/dto/ScholasticStanding.dto.ts

```diff
@@ -1,6 +1,9 @@
 export const APPLICATION_NOT_FOUND = "APPLICATION_NOT_FOUND";
 export const INVALID_OPERATION_IN_THE_CURRENT_STATUS =
   "INVALID_OPERATION_IN_THE_CURRENT_STATUS";
+export const APPLICATION_CHANGE_NOT_ELIGIBLE =
```

Comment:
There is nothing wrong with this particular code but we have been creating these error constants in different places.
We do have a file called `error-code.constants.ts` on both API and WEB project, can we have an agreement to use these files from now on and move the constants whatever is possible?
@guru-aot @ann-aot @dheepak-aot?

Reply:
yup, i too agree